_data/ome2021.json

```diff
@@ -96,7 +96,16 @@
       "workFeatured": [
         {
           "@type": "VideoObject",
-          "name": "Demo Introductions",
+          "name": "Community Meeting Introductions (3 videos, playlist length: 16min)",
```

Comment:
Aha, but hardcoded. Perhaps you can paste how you used the app so we can repeat what you did?

Reply:
It looks like there is a way using the Youtube API but I did not really have the time to investigate.

The workflow is pretty straightforward albeit manual: go to the URL, paste the full URL or the ID of the playlist and click on `Get Length`
![Screenshot 2021-06-03 at 14 43 07](https://user-images.githubusercontent.com/1355463/120654892-055d8000-c47a-11eb-924e-0563ac72429e.png)